app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -369,9 +367,8 @@ class ExplorationActivityTest {
     explorationDataController.stopPlayingExploration()
   }
 
-  // TODO (#1855): Resolve ktlint max line in app module test
   @Test
-  fun testAudioWithCellular_openRatioExploration_clickAudioIcon_clickNegative_checkAudioFragmentIsHidden() { // ktlint-disable max-line-length
+  fun testAudioWithCell_openRatioExploration_clickAudio_clickNeg_checkAudioFragmentIsHidden() {
```

Comment:
Here & elsewhere: you can probably use a different format to be clearer & avoid shortening pieces that can hurt readability:

``testActivity_ratioExp_withCellular_clickPlayAudio_clickNegative_audioFragmentIsHidden``

Note that you do have 100 characters to fit this in, so it doesn't need to be super short. :)

Ditto elsewhere in this suite.

Reply:
Got it! 
I have made changes everywhere so as to not abbreviate words.